File: customdialog/src/main/java/com/dunzhixuan/customdialog/MainActivity.java
package com.dunzhixuan.customdialog;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);

		GuideProtocolDialog dialog = new GuideProtocolDialog(this,R.style.lib_framework_dialog_style);
		dialog.setContentView(R.layout.m_guide_protocol_view);
		dialog.show();
	}
}

File: toastproblem/src/main/java/com/dunzhixuan/toastproblem/MainActivity.kt
package com.dunzhixuan.toastproblem

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import android.widget.Toast.LENGTH_SHORT

class MainActivity : AppCompatActivity() {

    lateinit var btn : Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn = findViewById(R.id.btn_toast)

        btn.setOnClickListener { Toast.makeText(this,"123", LENGTH_SHORT).show() }
    }
}
